pages/docs/2_modify/advanced/extending_local_version_comunica.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Linking local Comunica versions to other projects'
+description: 'Guide on how to use a local development version of Comunica with another local project'
+---
+
+In cases where a local development version of Comunica is consumed as a dependency of another project, linking the local development version of Comunica to the project is required.
+
+For example, various extensions of Comunica exist. These extensions utilize packages from the base Comunica framework and add additional packages or engine configurations. When working on local changes to base Comunica and needing to use these changes in an extension, you must link the local development version of  Comunica with the extension project. This page introduces a method to achieve this, though other methods are possible.
+
+This method does not require publishing the development version of Comunica to NPM, thus it is useful for testing changes before they are made public.
+
+## Lerna-linker
+
+Lerna-linker is a script designed to facilitate package linking in a Lerna monorepo. It iterates over all packages, executing `yarn unlink` and `yarn link` on each. It then saves all linked packages and runs `yarn link <package>` for each linked package in the Comunica extension.
+
+### Installation
+
+Install the script globally using the following:
+
+```bash
+$ npm install -g lerna-linker
```

Comment:
The npm readme currently says that it's used to link a local comunica version.
Does this mean it's hardcoded to comunica? Or does it also work for other monorepos?
If the latter, I'd make this more generic in the readme.

Reply:
It was hardcoded for comunica due to looking for all packages starting with "@", but I changed it and the README to be more general.